signbank/dictionary/models.py

```diff
@@ -782,12 +787,45 @@ def get_senses_with_similar_sensetranslations_dict(self):
                 similar_senses.append(sensedict)
         return sorted(similar_senses, key=lambda d: d['inglosses']) 
 
+    def ordered_examplesentences(self):
+        "Return a properly ordered set of examplesentences"
+        return self.exampleSentences.all().order_by('senseexamplesentence__order')
+
+    def reorder_examplesentences(self):
+        "when an examplesentence is deleted, they should be reordered"
+        for sentence_i, sentence in enumerate(self.ordered_examplesentences().all()):
+            print(sentence)
+            sensesentence = SenseExamplesentence.objects.all().get(sense=self, examplesentence=sentence)
+            sensesentence.order = sentence_i+1
+            sensesentence.save()
+
     def __str__(self):
         """Return the string representation of the sense, separated by | for every sensetranslation"""	
         str_sense = []
         for sensetranslation in self.senseTranslations.all():
             str_sense .append(str(sensetranslation))
         return " | ".join(str_sense)
+    
+class SenseExamplesentence(models.Model):
+    """An examplesentence belongs to one or multiple senses"""
+
+    sense = models.ForeignKey(Sense, on_delete=models.CASCADE)
+    examplesentence = models.ForeignKey(ExampleSentence, on_delete=models.CASCADE)
+    # order = models.IntegerField(
+    #     verbose_name    = _(u'Order'),
+    #     help_text           = _(u'What order to display this examplesentence within the sense.'),
+    #     default = 1
+    # )
```

Comment:
I just tried to migrate 0076 and 0077 with these line uncommented and that works without problems.